docs/fast_data/inputs_and_outputs.md

```diff
@@ -255,96 +291,145 @@ Delete operation:
 {
   "key": {
     "USER_ID": 123,
-    "FISCAL_CODE": "ABCDEF12B02M100O"
+    "TAX_CODE": "ABCDEF12B02M100O"
   },
   "value": {
     "op": "d",
     "before": {
       "USER_ID": 123,
-      "FISCAL_CODE": "the-fiscal-code-123",
+      "TAX_CODE": "the-fiscal-code-123",
       "COINS": 300000000
     },
     "after": null
   },
   "timestamp": "1234556789",
-  "offset": "100"
+  "offset": 100
 }
 ```
 
 #### Topic naming convention
-
-**producer**: the system producing its own events
-
 `<tenant>.<environment>.<source-system>.<projection>.ingestion`
```

Comment:
We should notify that this is the suggested convention (and default one when creating projections via Console)

Reply:
I would keep it as an affirmation to have as much people as possible follow our standards, if they do they can benefit from the defaults created by the console and whatever automatisms we will create with time. If we put suggested maybe more people won't bother following them and we may incur to bugs (like the problems we had with the pr-updates)